packages/renderer/src/lib/kube/KubePlayYAML.svelte

```diff
@@ -57,8 +60,24 @@ async function playKubeFile(): Promise<void> {
     if (userChoice === 'podman') {
       try {
         const result = await window.playKube(kubernetesYamlFilePath, selectedProvider);
+
         // remove the null values from the result
         playKubeResultRaw = JSON.stringify(removeEmptyOrNull(result), null, 2);
+        playKubeResultJSON = JSON.parse(playKubeResultRaw);
+
+        // If there are container errors, that means that it was *able* to create the container
+        // but if failed to start. We will add this to the "warning" section as we were able to create the
+        // We add this with comma deliminated errors
+        if (playKubeResultJSON.Pods.length > 0) {
+          const containerErrors = playKubeResultJSON.Pods.filter(pod => pod.ContainerErrors.length > 0);
```

Comment:
I got an error here. pod.ContainerErrors was undefined
```
error playing kube file TypeError: Cannot read properties of undefined (reading 'length')
    at KubePlayYAML.svelte:72:93
    at Array.filter (<anonymous>)
    at playKubeFile (KubePlayYAML.svelte:72:59)
```

Reply:
Ah thank you! I'll fix this. We should check to see that containerErrors actually exists before filtering.